site/docs/model-audit/usage.md

```
@@ -405,6 +405,34 @@ Automatic protection in archives:
 
    Note: ModelAudit automatically detects the actual format of `.bin` files and applies the appropriate scanner.
 
+6. **Disk Space Issues**
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)